src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -127,14 +131,15 @@ class AuthScreens extends React.Component {
         });
 
         // If we are on this screen then we are "logged in", but the user might not have "just logged in". They could be reopening the app
-        // or returning from background. If so, we'll assume they have some app data already and we can call reconnectApp() instead of openApp().
+        // or returning from background. If so, we'll assume they have some app data already and we can call openApp(true) instead of openApp().
```

Comment:
This change needs to be reverted. After this PR, `openApp` won't have any param.

Reply:
fixed comment